build.sh

```diff
@@ -0,0 +1,3 @@
+set -x
```

Comment:
You probably want the first line to be `#!/bin/bash`, then if you want `-x`, change it to `#!/bin/bash -x`